src/css/style.css

```diff
@@ -0,0 +1,574 @@
+:root {
+  --primary: #6167f2;
+  --primary-txt: #1f2937;
+  --secondary-txt: #1f2937;
+  --pri-bd: #bc6ce9;
+
+  /* --tertiary: #; */
+
+  /* --background: #; */
+
```

Comment:
- [ ] [OPTIONAL] Kindly remove unused code in order to have a cleaner code-base

Reply:
haha... Tolu my man..Thanks